tests/usb/usb2/test_request.py

```diff
@@ -1,24 +1,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-
 from sol_usb.gateware.usb.usb2.request import USBSetupDecoder
 from sol_usb.gateware.usb.usb2         import USBSpeed
 from sol_usb.gateware.test             import usb_domain_test_case
+from sol_usb.gateware.usb.usb2         import USBSpeed
+from sol_usb.gateware.usb.usb2.request import USBSetupDecoder
```

Comment:
These appear to duplicate the earlier imports? Did some line deletions not get staged?